storage/aptosdb/src/db/include/aptosdb_writer.rs

```diff
@@ -516,6 +519,39 @@ impl AptosDB {
         Ok(root_hash)
     }
 
+    fn commit_transaction_auxiliary_data(
+        &self,
+        txns_to_commit: &[TransactionToCommit],
+        first_version: u64,
+    ) -> Result<()> {
+        let _timer = OTHER_TIMERS_SECONDS
+            .with_label_values(&["commit_transaction_auxiliary_data"])
+            .start_timer();
+
+        let batch = SchemaBatch::new();
+        let num_txns = txns_to_commit.len();
+        txns_to_commit
+            .par_iter()
+            .with_min_len(optimal_min_len(num_txns, 128))
+            .enumerate()
```

Comment:
This won't work. SchemaBatch has a single Mutex in it. 
See how it's done in `transaction_db.commit_transactions()`

Reply:
no? it's used in save transactions and it's not done the same way?